testing/Class1.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace Bearded.Utilities.Testing
+{
+    public class Class1 { }
```

Comment:
New phone, who dis?

Reply:
I wasn't sure if the project would even build without any classes, and I wanted to keep the project creation PR separate from the implementation of the first helper /shrug

Reply:
Or maybe the builder doesn't care...